packages/react/src/transactions/useSetProfileMetadata.ts

```diff
@@ -87,12 +87,8 @@ export function useSetProfileMetadata(): UseDeferredTask<
 
   return useDeferredTask(async ({ metadataURI }: UseSetProfileMetadataArgs) => {
     invariant(
-      session?.authenticated,
-      'You must be authenticated to set profile metadata. Use `useLogin` hook to authenticate.',
-    );
-    invariant(
-      session.type === SessionType.WithProfile,
-      'You must have a profile to set profile metadata.',
+      session?.type === SessionType.WithProfile,
+      'You must be authenticated with a profile to block a profile. Use `useLogin` hook to authenticate.',
```

Comment:
I think this message is in the wrong hook here, did you intend to change the logic in `useSetProfileMetadata` hook?

Reply:
ah good spot, thanks!